src/ingest_validation_tools/upload.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         offline: bool = False,
         ignore_deprecation: bool = False,
         extra_parameters: Union[dict, None] = None,
-        token: str = "",
+        airflow_token: str = "",
```

Comment:
Not sure - but it might make most sense to call this globus_token or groups_token since that's really what the auth_tok is.